src/server/pps/server/api_server.go

```diff
@@ -1303,18 +1303,26 @@ func (a *apiServer) GetLogs(request *pps.GetLogsRequest, apiGetLogsServer pps.AP
 	pachClient := a.env.GetPachClient(apiGetLogsServer.Context())
 	ctx := pachClient.Ctx() // pachClient will propagate auth info
 
-	// Authorize request and get list of pods containing logs we're interested in
-	// (based on pipeline and job filters)
-	var rcName, containerName string
+	logCh := make(chan *pps.LogMessage)
+	var eg errgroup.Group
```

Comment:
That's correct, with a couple of notes:

1) There is no `followLogsForStats`, and no special logic within `getLogsForStats` for follow. This holds true for master as well. I think I verified that, despite this, stats w/ follow works as expected, but would want to double-check on that before merging.

2) The original idea behind this error group was to avoid the bug where one of those methods forgets to close `logCh`, but in retrospect that seems like overkill, so I can remove it.

Reply:
:+1: I this it's correct that there's no `followLogsForStats`. That code works by reading the logs from completed jobs from finished stats commit. Because it only works on jobs where the job is done and the stats commit is closed, you effectively can't use it to follow a job (which, for "follow" to be meaningful, must still be emitting logs and therefore *not* done)